libnetwork/resolvconf/resolvconf_unix_test.go

```diff
@@ -6,9 +6,13 @@ import (
 	"bytes"
 	"os"
 	"testing"
+
+	"github.com/docker/docker/testutil"
 )
 
 func TestGet(t *testing.T) {
```

Comment:
`/etc/resolv.conf` is usually 777, so I'd expect this to pass.  Maybe this is failing in your build environment because the file doesn't exist?

Reply:
stat shows it at 0644 in the mock chroot. The test fails for a missing file, however.

```
=== RUN   TestGet
    resolvconf_unix_test.go:15: open /etc/resolv.conf: no such file or directory
--- FAIL: TestGet (0.00s)
```

Reply:
If it's failing because of a missing file, `SkipWhenUnprivileged` isn't the appropriate condition.  You can check whether that file exists and skip for that reason instead.

Reply:
Makes sense to me. I'm going to drop this skip, and we can address this in a different PR.